examples/powertools-examples-core/cdk/infra/cdk.json

```diff
@@ -0,0 +1,53 @@
+{
+  "app": "mvn -e -q compile exec:java",
```

Comment:
not sure that make sense... exec:java is to execute the app, but we cannot execute the function like this...

Reply:
To the best of my understanding, this is the entrypoint for the CdkApp, not for the HelloWorld function.
And it's a required key